### PR TITLE
Reorganise and update content according to discussed plan

### PR DESCRIPTION
**Pull request by andydotxyz**
As the title says.
Rejigging content to have more at the top level nav. No more tour specific pages, just content.
Additionally tutorials are no longer separate - most went to "Extending Fyne" but not all.

Overall nav now is:
devnav

### CODE REVIEW

**Comment by andydotxyz** [comment]
I have managed to get it up and running temporarily for people without jekyll locally: http://18.134.241.169:4000/

**Review by Jacalz — CHANGES_REQUESTED** [review]
Here are some suggestions.

Exploring Fyne -> Widgets gets you a 404.

### Some other stuff while at it (suggestions):
I also think the testing tutorial needs to be updated now that the calculator isn't an example any more. FreeBSD arm64 also needs to be added to cross-compiling.

This was all of what I found. Looking really good.

File: started/packaging.md [hunk 1]
```diff
@@ -43,3 +43,22 @@ fyne install -icon myapp.png
 All of these commands also support a default icon file of `Icon.png` so that you
 can avoid typing the parameter for each execution.
 
+// TODO merge this in
```
[the remark this answers is not in the document]
Comment (andydotxyz) [reply started/packaging.md]: Thanks for catching this! document elements merged as I'd meant to do :)

File: index.md [hunk 2]
```diff
@@ -12,19 +12,29 @@ leading cross platform apps.
 
 <div class="intro-row">
     <div class="col-lg-4 text-center">
-        <p>To explore the design of our APIs without needing to set up a development environment we have a helpful tour available.</p>
+        <p>If you can't wait to start coding your first Fyne app then you can dive right in at our developer documentation.</p>
```
[the remark this answers is not in the document]
Comment (stuartmscott) [reply index.md]: I think the word 'coding' misleads the reader - if someone wanted to start coding, they would fire up an editor and start typing, not dive into documentation

[the remark this answers is not in the document]
Comment (andydotxyz) [reply index.md]: Fixed

**Review by Jacalz — APPROVED** [review]
This looks good to me now. The sheer amount of changes makes it a bit hard to review but I have tried to look at everything to the best of my ability.

**Pull request merged**